packages/daimo-api/package.json

```diff
@@ -18,7 +18,6 @@
   "repository": "https://github.com/daimo-eth/daimo.git",
   "dependencies": {
     "@coinbase/cbpay-js": "^1.8.0",
-    "@daimo/bulk": "^0.3.1",
```

Comment:
RIP for now

Reply:
what was daimo bulk? contracts for handling ops with the old entrypoint?